packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -391,128 +411,43 @@ export default async function main(
       }
 
       if (purchase === false || purchase instanceof UserAborted) {
-        handleCreateDeployError(output, firstDeployCall);
+        handleCreateDeployError(output, deployment);
         return 1;
       }
 
       handleCreateDeployError(output, purchase);
       return 1;
     }
 
-    if (handleCertError(output, firstDeployCall) === 1) {
-      return 1;
-    }
-
     if (
-      firstDeployCall instanceof DomainNotFound ||
-      firstDeployCall instanceof DomainNotVerified ||
-      firstDeployCall instanceof DomainPermissionDenied ||
-      firstDeployCall instanceof DomainVerificationFailed ||
-      firstDeployCall instanceof SchemaValidationFailed ||
-      firstDeployCall instanceof InvalidDomain ||
-      firstDeployCall instanceof DeploymentNotFound ||
-      firstDeployCall instanceof BuildsRateLimited ||
-      firstDeployCall instanceof DeploymentsRateLimited ||
-      firstDeployCall instanceof AliasDomainConfigured ||
-      firstDeployCall instanceof MissingBuildScript ||
-      firstDeployCall instanceof ConflictingFilePath ||
-      firstDeployCall instanceof ConflictingPathSegment
+      err instanceof DomainNotFound ||
+      err instanceof DomainNotVerified ||
+      err instanceof DomainPermissionDenied ||
+      err instanceof DomainVerificationFailed ||
+      err instanceof SchemaValidationFailed ||
+      err instanceof InvalidDomain ||
+      err instanceof DeploymentNotFound ||
+      err instanceof BuildsRateLimited ||
+      err instanceof DeploymentsRateLimited ||
+      err instanceof AliasDomainConfigured ||
+      err instanceof MissingBuildScript ||
+      err instanceof ConflictingFilePath ||
+      err instanceof ConflictingPathSegment
     ) {
-      handleCreateDeployError(output, firstDeployCall);
+      handleCreateDeployError(output, err);
       return 1;
     }
 
-    deployment = firstDeployCall;
-
-    if (now.syncFileCount > 0) {
-      const uploadStamp = stamp();
-
-      await new Promise((resolve, reject) => {
-        if (now.syncFileCount !== now.fileCount) {
-          debug(`Total files ${now.fileCount}, ${now.syncFileCount} changed`);
-        }
-
-        const size = bytes(now.syncAmount);
-        syncCount = `${now.syncFileCount} file${now.syncFileCount > 1
-          ? 's'
-          : ''}`;
-        const bar = new Progress(
-          `${chalk.gray(
-            '>'
-          )} Upload [:bar] :percent :etas (${size}) [${syncCount}]`,
```

Comment:
Is the progress bar still printed?

Reply:
No, this is a mistake. I was moving it to a different spot but lost it. Fixing 🙏🏻